package-lock.json

```diff
@@ -2683,8 +2683,8 @@
           "integrity": "sha1-ay6YIrGizgpgq2TWEOzK1TyyTQ8=",
           "dev": true,
           "requires": {
-            "path-exists": "^2.0.0",
-            "pinkie-promise": "^2.0.0"
+            "path-exists": "2.1.0",
```

Comment:
Apparently, 2-levels deep `requires` have switched back to be the resolved version rather than the originally requested version.

Would you mind if I revert all those changes and only keep the `three` (top-level) change? Or would you rather just commit things as-is and wait for an update of NPM (another) to fix things (again) ?

(and yes, this is really annoying…)

Reply:
OK, so, I started back from the `package-lock.json` from `mater` and ran `npm install`, and it produced the new file with only the 3-lines change for three.

So I'd bet you weren't actually using npm 6.2.0 when doing the `npm install`…

Reply:
Yeah it seems so, I'll try to see what's wrong on my computer. Thanks !